source/fundamentals/enterprise-authentication.txt

```diff
@@ -297,6 +297,10 @@ see the corresponding syntax.
       The following code example shows how to specify Azure IMDS OIDC authentication. 
       Replace the ``<percent-encoded audience>`` placeholder with the percent-encoded
       value of the ``audience`` parameter configured on your MongoDB deployment. 
+
+      You can't pass values containing the comma (``,``) character to the ``authMechanismProperties`` 
```

Comment:
Small suggestion: Although contractions are allowed by our style guide, it's also [noted here](https://www.mongodb.com/docs/meta/style-guide/style/contractions/#contractions) that if you want to emphasize the negative, don't use contractions. I think here we want to emphasize the negative
```suggestion
      You cannot pass values containing the comma (``,``) character to the ``authMechanismProperties`` 
```

Reply:
That makes sense, will update!